<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reAcselSave</class>
 <widget class="QDialog" name="reAcselSave">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Save Automatic Preset</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="6">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>All selected materials will be saved </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QFrame" name="x">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Name for this preset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="teSetName"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="4" colspan="2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Author</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="author"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="6">
    <widget class="QFrame" name="y">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Description for the preset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="description">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>90</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="defaultConfig">
     <property name="toolTip">
      <string>If enabled, this material configuration will be applied to the selected model regardless of what materials are used in the host application</string>
     </property>
     <property name="text">
      <string>This is a generic preset</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QCheckBox" name="setMaterialTypeOnly">
     <property name="toolTip">
      <string>If enabled, this material configuration will be applied to the selected model regardless of what materials are used in the host application</string>
     </property>
     <property name="text">
      <string>Only change the material type</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="btnReset">
     <property name="text">
      <string>Reset</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>308</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QPushButton" name="btnOK">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QPushButton" name="btnCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>x</zorder>
  <zorder>label_5</zorder>
  <zorder>defaultConfig</zorder>
  <zorder>y</zorder>
  <zorder>btnOK</zorder>
  <zorder>btnCancel</zorder>
  <zorder>btnReset</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>setMaterialTypeOnly</zorder>
 </widget>
 <tabstops>
  <tabstop>defaultConfig</tabstop>
  <tabstop>author</tabstop>
  <tabstop>teSetName</tabstop>
  <tabstop>description</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnOK</sender>
   <signal>clicked()</signal>
   <receiver>reAcselSave</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>reAcselSave</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
